src/js/percentage.js

```diff
@@ -0,0 +1,54 @@
+/*!
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("gpii.chartAuthoring.percentage");
+
+    gpii.chartAuthoring.percentage.calculate = function (value, total) {
+        value = parseFloat(value, 10);
```

Comment:
Would be nice to have another argument for the number of decimal places and have it default to 0 or 2 so the returned percentage is more controllable.